### PR TITLE
[BREAKING] feat: decouple PPO epochs from critic

### PR DESCRIPTION
**Pull request by emergenz**
### Checklist Before Starting

- [X] Search for similar PR(s).

### What does this PR do?

Currently, `ppo_epochs` is used for both the actor, as well as the critic. While this is the case for many PPO implementations (https://github.com/EdanToledo/Stoix, https://github.com/openai/baselines), there is no inherent reason why this should be the case. This is probably due to historic reasons, since OpenAI's PPO implementation had a single loss for both actor and critic (https://github.com/openai/baselines/blob/ea25b9e8b234e6ee1bca43083f8f3cf974143998/baselines/ppo2/model.py; inherent coupling of `num_epochs` between actor and critic).

This PR simply decouples the actor and critic epochs.

### API

The only API change is that I renamed `critic.ppo_epochs` to `critic.epochs` to reflect the new behaviour.

### Additional Info.

- **Training**: [FSDP, Megatron]
- **Inference**: []

### Checklist Before Submitting

- [X] Read the [Contribute Guide](https://github.com/volcengine/verl?tab=readme-ov-file#contribution-guide).
- [X] Apply [pre-commit checks](https://github.com/volcengine/verl?tab=readme-ov-file#code-linting-and-formatting).
- [X] Add `[BREAKING]` to the PR title if it breaks any API.
- [ ] Update the documentation about your changes in the [docs](https://github.com/volcengine/verl/tree/main/docs).

### CODE REVIEW

**Review by eric-haibin-lin — COMMENTED** [review]
the existing config still allows custom epoch num for critic via setting critic.ppo_epochs. does it not work for you?
